src/login/logind-dbus.c

```diff
@@ -4243,20 +4243,20 @@ int manager_start_scope(
                 const PidRef *pidref,
                 const char *slice,
                 const char *description,
-                char **wants,
-                char **after,
+                const char* const *requires,
+                const char* const *after,
```

Comment:
For the commit message: why? Please provide a short description of what this means, so that people can understand what changed without reading the whole patch.